student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(age, classroom, _name = 'Unknown', parent_permission: true)
   def play_hooky
     '¯(ツ)/¯'
   end
+
```

Comment:
I suggest you add an attr_reader for the classroom method to give this class the privilege to access it😉.

> `attr_reader :classroom`